INSTALL.md

```diff
@@ -63,38 +62,40 @@ checks locally before uploading your pull request.
 In order to install golangci-lint, you can run:
 
 ```
-$ curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh |
-  sh -s -- -b $(go env GOPATH)/bin v1.15.0
+# binary will be $(go env GOPATH)/bin/golangci-lint
+curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin vX.Y.Z
+
+# if GOPATH not set you can also install it into ./bin/ 
+curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s vX.Y.Z
+
 ```
 
 This will download and install golangci-lint from its Github releases
-page (using version v1.15.0 at the moment).
+page (using version v1.17.1 at the moment).
 
 ## Clone the repo
 
 Golang is a bit finicky about where things are placed. Here is the correct way
 to build Singularity from source:
 
 ```
-$ mkdir -p ${GOPATH}/src/github.com/sylabs && \
-  cd ${GOPATH}/src/github.com/sylabs && \
-  git clone https://github.com/sylabs/singularity.git && \
+$ git clone https://github.com/sylabs/singularity.git && \
```

Comment:
Would it also be worth adding this here? We download Golang to /tmp, should we also download singularity to /tmp?

```suggestion
cd /tmp
$ git clone https://github.com/sylabs/singularity.git && \

Reply:
well, is up to the reader... it can be in the preferred path, I do it at `/home/user/src` so, don't know will leave it to others to comment and approve

Reply:
yeah, we can change this later if needed, dont worry about it for now.